commit dfe8a86d42738052e91dc196aca3fc9c254bdfb2
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Fri Oct 6 13:23:32 2017 +0100

    brcmfmac: request_firmware_direct is quieter
    
    Since we don't have any CLM-capable firmware yet, silence the warning
    of its absence by using request_firmware_direct, which should also
    be marginally quicker.
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/drivers/net/wireless/broadcom/brcm80211/brcmfmac/common.c b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/common.c
index f0309e039592..b5889360da96 100644
--- a/drivers/net/wireless/broadcom/brcm80211/brcmfmac/common.c
+++ b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/common.c
@@ -194,7 +194,7 @@ int brcmf_c_process_clm_blob(struct brcmf_if *ifp)
 		return err;
 	}
 
-	err = request_firmware(&clm, clm_name, dev);
+	err = request_firmware_direct(&clm, clm_name, dev);
 	if (err) {
 		if (err == -ENOENT)
 			return 0;
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1129_dfe8a86d42738052e91dc196aca3fc9c254bdfb2.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1129_dfe8a86d42738052e91dc196aca3fc9c254bdfb2.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
